code/modules/antagonists/slaughter/slaughterevent.dm

```diff
@@ -3,7 +3,7 @@
 	typepath = /datum/round_event/ghost_role/slaughter
 	weight = 1 //Very rare
 	max_occurrences = 1
-	earliest_start = 36000 //1 hour
+	earliest_start = 15000 //30mins
```

Comment:
we have time helpers in https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/time.dm that exist specifically for this use. Use `30 MINUTES` etc, here and in other definitions

Reply:
Done.